src/components/Blabla/VideoPlayer/FacebookPlayer.js

```diff
@@ -91,6 +92,15 @@ class FacebookPlayer extends React.Component {
           this.createPlayer(videoId)
         }
       })
+
+      window.addEventListener(
+        "hashchange",
+        () => {
```

Comment:
It would be great if you can create a function  `seekToHash` Because you need to use the same function to remove the listener.

```
seekToHash = () => {
const hash = window.location.hash.substring(1) //Puts hash in variable, and removes the # character
this.onSeek(parseInt(hash, 10))
}
```
then you can use it here :
```
 window.addEventListener( "hashchange",this.seekToHash, false)
```

Reply:
I got You! changes coming through